docs/en/ingest-management/fleet/add-fleet-server-on-prem.asciidoc

```diff
@@ -0,0 +1,154 @@
+or deployed on-premises.
+// Need link to "minimum compute resource requirements"
+* The host meets the minimum compute resource requirements based on the maximum number
+of agents you plan to support in your deployment.
+
+// How do you set it up?
+To add a {fleet-server} host:
+
+. In {kib}, go to *Management* -> *{fleet}* -> *Settings*.
+For more information about these settings, see
+{fleet-guide}/fleet-settings.html[{fleet} settings].
+// lint ignore fleet-server
+. Under *Fleet Server hosts*, click *Edit hosts* and specify one or more host
+URLs your {agent}s will use to connect to {fleet-server}. For example,
+`https://192.0.2.1:8220`, where `192.0.2.1` is the host IP where you will
+install {fleet-server}. Save and apply your settings.
++
+TIP: If the **Edit hosts** option is grayed out, {fleet-server} hosts
+are configured outside of {fleet}. For more information, refer to
+{kibana-ref}/fleet-settings-kb.html[{fleet} settings in {kib}].
+
+. In the **{es} hosts** field, specify the {es} URLs where {agent}s will send data.
+For example, `https://192.0.2.0:9200`. Skip this step if you've started the
+{stack} with security enabled (you cannot change this setting because it's
+managed outside of {fleet}).
+
+. Save and apply the settings.
+
+[discrete]
+[[add-fleet-server-on-prem-add-server]]
+== Add {fleet-server}
+
+// What is it / why do you need it?
+// Prereqs
+
+// How do you set it up?
+Next, you'll add the {fleet-server}:
+
+. Click the **Agents** tab.
+. Click *Add {fleet-server}*.
+. This will open in-product instructions to add a {fleet} server using 
+one of two options: *Quick Start* or *Advanced*.
+* Use *Quick Start* if you want {fleet} to generate a
+{fleet-server} policy and enrollment token for you. The {fleet-server} policy
+will include a {fleet-server} integration plus a system integration for
+monitoring {agent}. This option generates self-signed certificates and is 
+*not* recommended for production use cases.
++
+[role="screenshot"]
+image::images/add-fleet-server.png[In-product instructions for adding a {fleet-server}]
+
+* Use *Advanced* if you want to either:
+** *Use your own {fleet-server} policy.* You can create a new {fleet-server}
+policy or select an existing one. Alternatively you can
+{fleet-guide}/create-a-policy-no-ui.html[create a {fleet-server} policy without using the UI],
+and then select the policy here.
+** *Use your own TLS certificates.* TLS certificates encrypt traffic between
+{agent}s and {fleet-server}. To learn how to generate certs, refer to
+{fleet-guide}/secure-connections.html[Configure SSL/TLS for self-managed {fleet-server}s].
++
+[NOTE]
+====
+If you are providing your own certificates:
+
+* Before running the `install` command, make sure you replace the values in
+angle brackets. 
+* Note that the URL specified by `--url` must match the DNS name used to
+generate the certificate specified by `--fleet-server-cert`.
+====
++
+// image ?
```

Comment:
I was a bit confused by this... what's the difference between just choosing "Quick start" at the top level vs. choosing "Advanced" then selecting the "Quick start" radio button? Are these actually the same or just have the same name?

<img width="684" alt="Screen Shot 2022-09-13 at 6 02 29 PM" src="https://user-images.githubusercontent.com/10479155/190512823-0ca5415e-aae3-48de-a944-aa4ca954dfa3.png">

Reply:
I think they are the same, but not sure. IMO this UI still needs a lot of work.